weblogic-azure-aks/src/main/bicep/modules/_deployment-scripts/_ds-datasource-connection.bicep

```diff
@@ -43,8 +41,35 @@ resource deploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
   identity: identity
   properties: {
     azCliVersion: azCliVersion
-    arguments: const_arguments
     environmentVariables: [
+      {
+        name: 'AKS_RESOURCE_GROUP_NAME'
+        value: aksClusterRGName
+      }
+      {
```

Comment:
Yes, this is much cleaner.  I have filed a [User Story](https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1590695) to apply this approach throughout the offer.